ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/MLSdkAsyncHttpResponseHandler.java

```diff
@@ -38,6 +39,7 @@
 
 @Log4j2
 public class MLSdkAsyncHttpResponseHandler implements SdkAsyncHttpResponseHandler {
+    public static final String AMZ_ERROR_HEADER = "x-amzn-ErrorType";
```

Comment:
what is this error about? Can we add more context?

Reply:
Please check the PR description, it has a lot of context in the aforementioned github issues.